### DISCUSSION TITLE
operator voor het omdraaien van de winding order

### DISCUSSION THREAD

**Discussion by jipclaassens** [ObjectVision/GeoDMS#63] [Ideas]
Maarten Hilferink [28 Feb 2022 at 17:09](https://trello.com/c/Ov4YGx7k/53-operator-voor-het-omdraaien-van-de-winding-order#comment-621cf3c628c2fe12a2dc3890)
wellicht ook al OK met split_polygon

[Reply](https://trello.com/c/Ov4YGx7k/53-operator-voor-het-omdraaien-van-de-winding-order#)
Maarten Hilferink [28 Feb 2022 at 17:08](https://trello.com/c/Ov4YGx7k/53-operator-voor-het-omdraaien-van-de-winding-order#comment-621cf38a0a150a42f2153ebc)
eerst bg_inflate, bg_intersect, remove_lakes, dan cleaning heroverwegen.

[Reply](https://trello.com/c/Ov4YGx7k/53-operator-voor-het-omdraaien-van-de-winding-order#)
Jip Claassens [28 Feb 2022 at 17:05](https://trello.com/c/Ov4YGx7k/53-operator-voor-het-omdraaien-van-de-winding-order#comment-621cf2d7ec98367313762cbb)
of een fix geometry

**Comment by MaartenHilferink** [comment]
Tevens meenemen: 
- toevoegen van een projection property dat een synoniem is van het huidige format
- rol van projection / EPGS bij base projections en/of gdal datasource; 
- operatoren om coordinaten te spiegelen:
  -  mirror_diagonal: (x, y) -> (y,x)
  - mirror_row: row(p) col(p) -> point_rowcol(-row(p), col(p))
  - mirror_col
  - mirror_first
  - mirror_second
- operatoren/ functies om coordinaten te checken
  -  warning op gebruik van base projection item zonder projection
  - error op inconsistentie tussen base projection item en gdal datasource
  - projection specifieke range checks op ingelezen/berekende items

**Comment by MaartenHilferink** [comment]
Zie ook: https://geodms.nl/mantis/view.php?id=335

**Comment by MaartenHilferink** [comment]
Excerpt from https://www.esri.com/content/dam/esrisites/sitecore-archive/Files/Pdfs/library/whitepapers/pdfs/shapefile.pdf page 8 section "Polygon":  ... The order of vertices or orientation for a ring indicates which side of the ring
is the interior of the polygon. The neighborhood to the right of an observer walking along
the ring in vertex order is the neighborhood inside the polygon. Vertices of rings defining
holes in polygons are in a counterclockwise direction. Vertices for a single, ringed
polygon are, therefore, always in clockwise order. The rings of a polygon are referred to
as its parts
